Content.Server/GameTicking/GameTicker.cs

```diff
@@ -67,7 +67,7 @@ public partial class GameTicker : GameTickerBase, IGameTicker
         public const float PresetFailedCooldownIncrease = 30f;
         private const string PlayerPrototypeName = "HumanMob_Content";
         private const string ObserverPrototypeName = "MobObserver";
-        private const string MapFile = "Maps/saltern.yml";
+        private const string MapFile = "Maps/stationstation.yml";
```

Comment:
🤔

Reply:
I was using it for testing no judgy pants here mister